src/test/java/org/folio/circulationbff/api/CirculationBffClaimItemReturnedApiTest.java

```diff
@@ -0,0 +1,128 @@
+package org.folio.circulationbff.api;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.equalTo;
+import static com.github.tomakehurst.wiremock.client.WireMock.jsonResponse;
+import static com.github.tomakehurst.wiremock.client.WireMock.postRequestedFor;
+import static com.github.tomakehurst.wiremock.client.WireMock.urlPathEqualTo;
+import static org.mockito.Mockito.when;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+import static org.springframework.http.MediaType.APPLICATION_JSON;
+import static wiremock.org.apache.hc.core5.http.HttpStatus.SC_NO_CONTENT;
+
+import java.util.Date;
+import java.util.UUID;
+
+import org.folio.circulationbff.domain.dto.ClaimItemReturnedRequest;
+import org.folio.circulationbff.domain.dto.UserTenant;
+import org.folio.circulationbff.service.TenantService;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+
+import com.github.tomakehurst.wiremock.client.WireMock;
+
+import lombok.SneakyThrows;
+
+class CirculationBffClaimItemReturnedApiTest extends BaseIT {
+
+  private static final String CLAIM_ITEM_RETURNED_URL = "/circulation-bff/loans/%s/claim-item-returned";
+  private static final String CIRCULATION_CLAIM_ITEM_RETURNED_URL = "/circulation/loans/%s/claim-item-returned";
+  private static final String TLR_CLAIM_ITEM_RETURNED_URL = "/tlr/loans/claim-item-returned";
+  private static final String REQUESTS_MEDIATED_CLAIM_ITEM_RETURNED_URL = "/requests-mediated/loans/%s/claim-item-returned";
+  @Mock
+  private TenantService tenantService;
+
+  @Test
+  @SneakyThrows
```

Comment:
We can place this annotation on `performClaimItemReturned` to avoid annotating every test